client/src/app.jsx

```diff
@@ -46,29 +57,65 @@ class App extends React.Component {
   }
 
   add() {
+    console.log('ADD', this.state.totalEnrolled);
     $.ajax({
       type: 'POST',
-      url: 'http://localhost:4000/api/addTitle',
+      url: 'http://localhost:3001/api/addTitle',
       data: {total: this.state.num},
       success: () => console.log('successfully made a post')
     });
   }
 
+
   render() {
     return (
       <div>
-        <h4>Enter number of titles to populate</h4>
+        {/* <h4>Enter number of titles to populate</h4>
         <input value = {this.state.num} onChange={this.onChange} />
-        <button onClick={this.add}> Add Titles </button>
+        <button onClick={this.add}> Add Titles </button> */}
+        <div>
+          <h1 className="banner-title">
+            <Titles title={this.state.titles}/>
+          </h1>
+        </div>
+        <span className="side-bar">
+          <span className="offered">Offered By: </span>
+          <span className="university">Stanford</span>
```

Comment:
Once we get everything up and running, you can get this data from the Instructors Service.  The backend should be complete for that, but I have a few changes to make to the seed script once we get to coding today.

Reply:
thanks Jay, yeah this week, planning to incorporate all the async requests to everyone's services